app/main/helpers/frameworks.py

```diff
@@ -122,3 +123,127 @@ def count_drafts_by_lot(drafts, lot):
     return len([
         draft for draft in drafts if draft['lot'] == lot
     ])
+
+
+def get_statuses_for_lot(
+    has_one_service_limit,
+    drafts_count,
+    complete_drafts_count,
+    declaration_status,
+    framework_status,
+    lot_name,
+    unit,
+    unit_plural
+):
+
+    if not drafts_count and not complete_drafts_count:
+        return []
+
+    framework_is_open = 'open' == framework_status
```

Comment:
This kind of thing is where Java-style brackets win for clarity.  Don't know if PEP8 will even allow them though.